styles.css

```diff
@@ -48,7 +69,25 @@ p {
 }
 
 .navbar {
-  padding: 7% 10% 0% 0%;
+  margin: 5% 10% 0% 0%;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.bar {
+  display: block;
+  width: 1rem;
+  height: 0.03rem;
+  margin: 10%;
+  background-color: #28352f;
+  border: 1px solid #28352f;
```

Comment:
[OPTIONAL] The previous reviewer requested the change to the close button color but the color he asked for was not the correct one, so I will keep this optional, kindly use the color defined on the Figma template. You can find the correct color below on the screenshot. 👍 
<img width="490" alt="Screenshot 2023-03-06 at 8 21 32 PM" src="https://user-images.githubusercontent.com/25887253/223186419-d4015d16-1e5c-444f-8628-146b5175afed.png">

Reply:
Your feedback is progressive for me :) But I have not used the close button instead I used CSS styling and hamburger for developing the cross-icon that's why it is technically impossible to give the cross-icon the same colour. Still, I'll look into it and make changes accordingly. 👌 @huseyinbiyik